### DISCUSSION TITLE
Greek letters in column title

### DISCUSSION THREAD

**Discussion by mjroux69** [rstudio/gt#941] [Q&A] [answered]
I'm trying to format the column titles in a gt table, to 
1) include greek letters (more specifically the mu, µ)
The encoding is set to UTF-8 in the R Studio options.

As mu is directly accessible on French keyboards, I initially included it directly in the string 
`cols_label(total_length = "Total length (µm)")`
However, in the output, it is replaced by a white question mark in a black diamond.
The second attempt was to use markdown formatting
`cols_label(total_length = md("Total length ($\mu$m)")`
_Error: '\m' is an unrecognized escape in character string starting ""Total length ($\m"_
Doubling the dollar signs brought a similar error message:
_Error: '\m' is an unrecognized escape in character string starting ""Total length ($$\m"_

With not much hope, as it should lead to the same output as the first syntax, I tried
`cols_label(total_length = paste("Total length (", intToUtf8(0xB5), "m)", sep =""))`
and indeed it produced the same output with the white question mark in a black diamond.

What is the way to go ?

2) Include superscript
This should be straightforward using `cols_label()` and `md()`.
However, while `md()` processes correctly the italic and bold tags, it does not process the `^`to render superscript:
Using `html()` and the <sup> </sup> tags works, but I was wondering why `md()` does not work here, while it works fine in ggplot.

Best regards,

Michel

**Comment by mjroux69** [comment, marked as answer]
I solved the mu issue by switching to `html()`with the &mu; code.

**Reply by alex-lauer** [reply]
Switching to unicode encoding worked for me:
```
library(gt)
data("exibble")

exibble %>% 
  gt() %>% 
  cols_label(num = "Total Length (\U03BCm)")
```

**Reply by mjroux69** [reply]
Thanks Alexandra.
I've tried your code and it works also for me.
Just by curiosity, I've also tried the code from my previous attempts
`cols_label(total_length = "Total length (µm)")` and it now works.
`cols_label(num = md("Total length (µm^2)"))` still does not convert ^2 into a superscript, 
but `cols_label(num = md("Total Length (µm<sup>2</sup>)"))`works fine.
May be this was fixed in the recent gt update ?
